scripts/fair-eva.py

```diff
@@ -201,6 +205,50 @@ def print_table(result_json, show_totals=False):
         print(table)
 
 
+def search(keytext):
+    args = get_input_args()
+    max_tries = 5
+
+    headers = {
+        "accept": "application/json",
+    }
+    good = 0
+    params = {"facets": "false", "q": keytext}
+    print(args.query)
+    response = requests.get(
+        "https://ics-c.epos-ip.org/development/k8s-epos-deploy/dt-geo/api/v1/resources/search",
```

Comment:
don't hardcode this value, get it from `args.repository` or better `metadata_endpoint`